README.md

```diff
@@ -99,9 +99,44 @@ would in a Django shell.
 
 ### Running tests
 
-Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
+### NOTE: These commands can be run with ```docker-compose exec``` to execute them in an already-running container, or with ```docker-compose run --rm``` to execute them in a new container.
 
-    docker-compose run --rm web pytest
+
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
+    
+    docker-compose exec web pytest
+    # Run Python tests in a single file
+    docker-compose exec web pytest /path/to/test.py
+    # Run Python test cases in a single file that match some function/class name
+    docker-compose exec web pytest /path/to/test.py -k test_some_logic
+    # Run Python linter
+    docker-compose run --rm web pylint
+    
+    # Run black commands to format the code
+    # first go into the web bash
+    docker-compose exec web bash
```

Comment:
Please remove this line and the comment above it, and change the commands/comments below to:

```bash
# Format all python files
docker-compose run --rm web black .
# Format a specific file
docker-compose run --rm web black /path/to/file.py
```